eng/pipelines/templates/variables/vmr-build.yml

```
@@ -62,6 +62,12 @@ variables:
 
 - name: almaLinuxX64Rid
   value: almalinux.8-x64
+- name: alpinelinuxX64Rid
```

I think it would be better to rename this to avoid confusion with the version specific rid variables.
```suggestion
- name: linuxMuslX64Rid
```

---

Updated